modules/docker/docker_compose.go

```diff
@@ -16,15 +16,20 @@ type Options struct {
 
 // RunDockerCompose runs docker-compose with the given arguments and options and return stdout/stderr.
 func RunDockerCompose(t testing.TestingT, options *Options, args ...string) string {
-	out, err := RunDockerComposeE(t, options, args...)
+	out, err := RunDockerComposeE(t, false, options, args...)
 	if err != nil {
 		t.Fatal(err)
 	}
 	return out
 }
 
+// RunDockerComposeStdoutE runs docker-compose with the given arguments and options and return only stdout.
+func RunDockerComposeStdOutE(t testing.TestingT, options *Options, args ...string) (string, error) {
+	return RunDockerComposeE(t, true, options, args...)
+}
+
 // RunDockerComposeE runs docker-compose with the given arguments and options and return stdout/stderr.
-func RunDockerComposeE(t testing.TestingT, options *Options, args ...string) (string, error) {
+func RunDockerComposeE(t testing.TestingT, stdout bool, options *Options, args ...string) (string, error) {
```

Comment:
I think this should be an internal helper function that `RunDockerComposeE` calls under the hood, for backwards compatibility and consistency with the `RunDockerCompose` function.

Reply:
Like this? https://github.com/gruntwork-io/terratest/pull/665/commits/4fe3a995298e5d284de0f48b239b5a78791f831a

Reply:
Yup!